teacher.rb

```diff
@@ -1,10 +1,10 @@
-require_relative '/person'
+require_relative 'person'
 
 class Teacher < Person
   attr_accessor :specialization
 
-  def initialize(specialization, age, name = 'unknown', parent_permission: true)
-    super(age, name, parent_permission: parent_permission)
+  def initialize(age, specialization, name, parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] As the previous reviewer required, there are some mistakes we can find on the displaying the people. For example,  the teacher has a name shows  her/his specialization rather than the actual name. Also, I recommend you to fix the student's 'parent_permission' column which should only show true or false. (But it's not in the actual design. You can remove it.)
![image](https://github.com/Kgomotso196/OOP-School-Library/assets/91486791/da7f6a4e-8775-4120-91a2-53613313b501)